src/distance.hpp

```diff
@@ -36,15 +38,30 @@ class DistanceDeterminationAlgorithm {
 */
 class SphericalDistanceDeterminationAlgorithm : public DistanceDeterminationAlgorithm {
 public:
-    SphericalDistanceDeterminationAlgorithm(float radius);
+    SphericalDistanceDeterminationAlgorithm(float radius, Camera &cam) : radius_(radius), cam_(cam) {};
+    /**
```

Comment:
Delete this comment.

its equivalent to just using the constructor inside the `{}`